r/R/dplyr-functions.R

```diff
@@ -848,6 +844,21 @@ nse_funcs$wday <- function(x,
   Expression$create("day_of_week", x, options = list(count_from_zero = FALSE, week_start = week_start))
 }
 
+nse_funcs$is.Date <- function(x) {
+  inherits(x, "Date") ||
+    (inherits(x, "Expression") && x$type_id() %in% Type[c("DATE32")])
+}
+
+nse_funcs$is.instant <- nse_funcs$is.timepoint <- function(x) {
+  inherits(x, c("POSIXt", "Date")) ||
```

Comment:
For exact parity with lubridate, this would be
```suggestion
  inherits(x, c("POSIXt", "POSIXct", "POSIXlt", "Date")) ||
```
(https://github.com/tidyverse/lubridate/blob/9cece5680d0280c5cd1ccdbc9309ccb4724b3a34/R/instants.r#L15)